IND/MarketIndices/BasketIndices.rdf

```diff
@@ -294,7 +295,7 @@
 	</owl:DatatypeProperty>
 	
 	<owl:ObjectProperty rdf:about="&fibo-ind-mkt-bas;hasDurationOfProtection">
```

Comment:
definition does not explain "protection".

Reply:
added "against default"

Reply:
Removed the property